.kitchen.yml

```diff
@@ -54,6 +54,12 @@ suites:
             - test/fixtures/simple_example/sshkey
           shell: true
           shell_options: "--login"
+
+        - name: org-iam
```

Comment:
Right now these tests will only run on the `simple-example` test fixture, which means we have a gap for shared-vpc. We should address this but that change will be somewhat involved, so this is mainly a note for posterity.

Reply:
The tests have been added to the `shared_vpc` suite.